package.json

```diff
@@ -5,11 +5,12 @@
   "scripts": {
     "api": "node tasks/build.js --api",
     "benchmark-server": "node tests/benchmark-server/index.js",
+    "build": "node tasks/build.js --build",
```

Comment:
`build.js --build`? It's quite common to have Release and Debug building schemes, so let's replace with `--release`.